ci/release/update-version.sh

```diff
@@ -46,11 +46,13 @@ function sed_runner() {
 }
 
 # setup update
-sed_runner 's/version="0.*"/version='"'${NEXT_FULL_TAG}'"'/g' setup.py
+#sed_runner 's/version="0.*"/version='"'${NEXT_FULL_TAG}'"'/g' setup.py
```

Comment:
Drop the commented lines if they don't work, otherwise looks good, thanks for this :)

Reply:
Done. Thanks.